Addressing Issue #1552 remove auto-pauses

CristianRos opened the pull request:
Hey, this is my first PR so I'm a little bit lost, I hope everything it's okay, tell me if I need to change anything :)

I felt fairly confident I could help with this issue in particular. But as I was approaching it I thought it could've been done in different ways:

![image](https://github.com/coppolaemilio/dialogic/assets/133830154/fcac7a59-01b7-4100-907e-b1099ba948ca)

I went with the 'D' case, with the 'B' one being the second I like the most. I feel like if the remove button is to the right of the time input I would delete the auto-pause set trying to change the value with the arrows.

Also while addressing this I realized 2 things:

- When you add a lot of auto-pauses the UI breaks, I don't think anyone is gonna need that much but maybe is worth to know: 
![image](https://github.com/coppolaemilio/dialogic/assets/133830154/f00ad6e7-5b92-4a84-86c1-de038e4ade16)

- It only saves the auto-pauses when swapping the main tabs:
![image](https://github.com/coppolaemilio/dialogic/assets/133830154/239b2826-4f26-476c-a543-1d2d5ac975ce)

I'm not sure if that's already known but I wanted to say them in case they aren't

Jowan-Spooner commented:
Hey, thanks for contributing. I think third option (text input, time input, remove button) makes the most sense. 

The autopauses only saving when switching tabs is by design but I would be fine with changing it. 

If you feel like it you can add a Scroll container into the settings page to keep it from breaking, but yeah, so many pauses are unlikely.

CristianRos commented:
I don't feel confident enough about touching the saving part, I mentioned it because I kind of thought it would save changes directly and I was struggling a bit thinking I introduced a bug or something like that haha

But I changed what you said, now the autopauses follow [text input, time input, remove button] and there is a scroll bar when you add a lot of them.

Hope that's okay!

Jowan-Spooner commented:
Looks great! Thanks!
